unifi_console/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "display_name": "Unifi_Console",
```

Comment:
```suggestion
  "display_name": "Unifi Console",
```

Reply:
To match the casing in the `public_title`

Reply:
This gets rejected by the CI